source/whats-new.txt

```diff
@@ -56,6 +57,44 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.5:
+
+What's New in 6.5
+-----------------
+
+The {+driver-short+} v6.5 release includes the following features:
+
+- Moved ``ObjectId`` generation to the ``pkFactory`` class for bulk write
+  operations.
+
+.. warning::
+
+   If you previously specified the ``pkFactory`` to handle bulk writes,
+   the ``_id`` fields of the documents inserted by using bulk writes may
+   be inconsistent with the behavior in this version.
+
+- Fixes the read preference sent with read operations when directly connected
+  to a secondary in the replica set to ``primaryPreferred``.
```

Comment:
```suggestion
- Sets the read preference that is sent with read operations to ``primaryPreferred`` when
  you are directly connected to a secondary in the replica set.
```

Reply:
Thanks for the rewording. I'll use parts of this suggestion, but will continue to use the verb "fixes" to make sure it's understood that way. From the release notes:

> Fixed applying read preference to commands depending on topology